scripts/release/check_sync_criteria.py

```diff
@@ -48,69 +43,12 @@ def main(argv=sys.argv[1:]):
     add_argument_cache_dir(parser, '/tmp/package_repo_cache')
     args = parser.parse_args(argv)
 
-    success = check_sync_criteria(
+    target = Target(args.os_name, args.os_code_name, args.arch)
+    success = partition_packages(
         args.config_url, args.rosdistro_name, args.release_build_name,
-        args.os_name, args.os_code_name, args.arch, args.cache_dir)
+        target, args.cache_dir)
     return 0 if success else 1
 
 
-def check_sync_criteria(
```

Comment:
Where have the checks of this function been moved to? I assume we still want to check `sync_packages` / `sync_package_count`.

Reply:
I was going to make it more strict with this requiring no regressions but we can do this one step at a time and not enforce that yet.

Reply:
This patch doesn't update any of the existing documentation and I don't think that kind of change (by default) is even desirable. These sync criteria are what is defined in the config and should continue to work as is. Therefore I would suggest to add a new option to optionally perform additional checks.